main/zoe/api/records.md

```diff
@@ -0,0 +1,151 @@
+# Records
+
+<Zoe-Version/>
+
+These records (JavaScript objects with data properties and no
+functions) are used throughout Zoe.
+
+## SeatAndPayout
+
+This is returned by a call to `redeem` on Zoe. A seat is an arbitrary object whose methods allow the user to take certain actions in a contract. The payout is a promise that resolves to an object which has keywords as keys and promises for payments as values. Note that while the payout promise resolves when an offer is completed, the promise for each payment resolves after the remote issuer successfully withdraws the payment.
```

Comment:
Need to explain keywords, and the syntactic requirements on keywords. If this is already explained elsewhere, link to it from here.